server/models/post.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const partitionPromise = require('@quarterto/partition-promise');
+const transform = require('../lib/transform');
+const articleModel = require('./article');
+const database = require('../lib/database');
+const getCanonical = require('./canonical');
+const fbApi = require('../lib/fbApi');
+const mode = require('../lib/mode');
+
+exports.get = async function get() {
+	const since = await database.getLastABCheck();
+	await database.setLastABCheck(Date.now()); // set this as soon as possible because this might take a while
```

Comment:
I wanted to avoid the case where it would miss a post made during the current check, but yeah it should handle that